src/app/admin-api/admin-api.component.ts

```diff
@@ -11,27 +11,27 @@ import { isValidPerson } from '../_validation/PersonValidation';
 })
 export class AdminAPIComponent implements OnInit {
   data: Array<PersonInput> = [
-    {id: 1, firstName: "Alek", lastName: "Kulinski", phone: "123123123", address:"Krk", message: ""},
-    {id: 2, firstName: "Przemek", lastName: "Asdd", phone: "123123123", address:"Krk", message: ""},
+    { id: 1, firstName: "Alek", lastName: "Kulinski", phone: "123123123", address: "Krk", message: "" },
+    { id: 2, firstName: "Przemek", lastName: "Asdd", phone: "123123123", address: "Krk", message: "" },
   ]
   formGroup = new FormGroup({
-    firstName: new FormControl(''),
+    firstName: new FormControl('', [Validators.required, isValidFirstName()]),
```

Comment:
what was the reason to execute the function, let's take a look at the Validators.required realization, it's passing VAlidatosFn directly, 
what do you think if we will make it in the same way, 
so isValidFirstName, instead of isValidFirstName()

Reply:
what we need to do for it:
`isValidFirstName:  () => ValidatorFn` ===> `isValidFirstName: ValidatorFn`